src/scripts/main.js

```diff
@@ -1,3 +1,36 @@
 'use strict';
 
-// write your code here
+const body = document.querySelector('body');
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise(resolve => {
+  logo.addEventListener('click', () => {
+    resolve(createDiv('Promise was resolved!'));
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(
+    () => reject(createDiv('Promise was rejected!', 'error-message')),
+    3000
+  );
+});
+
+promise1
+  .then()
+  .catch((err) => alert(err));
+
+promise2
+  .then((err) => alert(err))
```

Comment:
It's not an error, as then method executes only on success)
It's just not what we need, but it is certainly not an error